sdk/graphrbac/azure-graphrbac/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 0.61.2 (2024-10-31)
```

Comment:
You also need update version for https://github.com/Azure/azure-sdk-for-python/blob/5f729a78315f1afc1cc39dd375fabba53baa5394/sdk/graphrbac/azure-graphrbac/azure/graphrbac/version.py